### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vitoriaHeliane**
#### What is the purpose of this pull request?

As the title says 
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,32 +19,86 @@ A Menu Component. Shows a superior bar with links and dropdown menus.
...
 ```
+
+Also, create a `vtex.menu.css` file in `styles/css` for your handlers customization.
+
+
+## Troubleshooting
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/menu/issues). Also feel free to [open issues](https://github.com/vtex-apps/menu/issues/new) or contribute with pull requests.
+
+## Tests
+To execute our tests go to `react/` folder and run `npm t`
```
Comment (GustavoMA) [inline README.md]: run `yarn test` :hammer:

File: README.md [hunk 2]
```diff
@@ -15,32 +19,86 @@ A Menu Component. Shows a superior bar with links and dropdown menus.
 
 [![Build Status](https://travis-ci.org/vtex-apps/menu.svg?branch=master)](https://travis-ci.org/vtex-apps/menu)
```
Comment (GustavoMA) [inline README.md]: Remove this part of CI and Travis CI.

Comment (GustavoMA) [reply README.md]: Also, add the `See our LTS policy for more information.`. See minicart README exxample.

File: README.md (hunk 1 above)
Comment (vitoriaHeliane) [reply README.md]: This app do not use `yarn` to run the tests.

File: README.md (hunk 2 above)
Comment (vitoriaHeliane) [reply README.md]: The instruction is to keep it if the Travis CI is okay, so it should stay even if the minicart doc does not have it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Update the `README.md` with the docs.
```
Comment (paul0vinicius) [inline CHANGELOG.md]: This is no longer necessary once the docs will not be released.

File: README.md [hunk 4]
```diff
@@ -1,46 +1,155 @@
...
+
+The VTEX Menu app is store component that shows a bar with links and drop-down menus, and this app is used by store theme.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
 
 ## Release schedule
 | Release  | Status              | Initial Release | Maintenance LTS Start | End-of-life | Dreamstore Compatibility
 | :--:     | :---:               |  :---:          | :---:                 | :---:       | :---: 
 | [1.x]    | **Maintenance LTS** |  2018-08-17     | 2018-11-27            | March 2019  | 1.x
 | [2.x]    | **Current Release** |  2018-11-27     |                       |             | 2.x
```
Comment (paul0vinicius) [inline README.md]: Current release comes first

File: README.md [hunk 5]
```diff
@@ -1,46 +1,155 @@
...
 ```
 
-## Tests
+### Blocks API
+
+When implementing this app as a block, various inner blocks may be available. The following interface lists the available blocks within menu and describes if they are required or optional.
 
-Run the tests with the command
+```json
+{
```
Comment (paul0vinicius) [inline README.md]: There is no need for '{', once you are using the ```json syntax. So I think it's best to write it this way:
```json 
"menu-link": {
  "component": "index"
}
```

**Pull request merged**